doc/man3/OBJ_nid2obj.pod

```diff
@@ -155,9 +157,10 @@ a NID or B<NID_undef> on error.
 
 OBJ_add_sigid() returns 1 on success or 0 on error.
 
-OBJ_obj2txt() returns the number of bytes written to I<buf> if I<buf_len> is big enough. 
-Otherwise, the result is truncated and the total amount of space required is returned. 
-It also returns -1 on error.
+i2t_ASN1_OBJECT() an OBJ_obj2txt() return -1 on error.
+On success, they return the lentgh of the string written to I<buf> if I<buf> is
```

Comment:
Typo: length

Reply:
Fixed, and on this occasion squashed the commits.

Reply:
please push